VoteSystem.Console/Program.cs

```diff
@@ -169,24 +179,16 @@ static void Main(string[] args)
                                 }
                                 Console.WriteLine("Which rights do you want to give? (Admin/Access)");
                                 string answer_for_rights = Console.ReadLine();
-                                if (answer_for_rights == "Admin")
-                                {
-                                    Console.WriteLine("Enter email for user who you want to give policy:");
-                                    string email = Console.ReadLine();
-                                    User user = userRepository.GetUser(email);
-                                    managePolicy.GiveAdminPolicyToUser(user.Id, poll.Id);
-                                }
-                                else if (answer_for_rights == "Access")
-                                {
-                                    Console.WriteLine("Enter email for user who you want to give policy:");
-                                    string email = Console.ReadLine();
-                                    User user = userRepository.GetUser(email);
-                                    managePolicy.GivePolicyToUser(user.Id, poll.Id);
-                                }
-                                else
-                                {
+                                if(!Enum.TryParse(answer_for_rights, out PolicyType policyType)) {
```

Comment:
here is important thing:
1 - capture input 
2 - pass it to service
1 - capture input 
2 - pass it to service 
e.t.c.

What "was" here is you've mixed up input capturing and passing it to services.
`string answer_for_rights = Console.ReadLine();` was written in the 171 line, while response to incorrect input was written in 188 line, after multiple calls to services and two different branches.

That is messing the aforementioned concept and makes it nearly-impossible to extract "logic" from "view".

I've refactored the code so input is gathered in 180-188 and then service is called at 190-191